### ISSUE TITLE
Pass `-f` to frontend/Dockerfile `rm` run so that missing files do not cause errors

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse#2040]
## Problem

It isn't possible to run the frontend Dockerfile build locally any more because it assumes that it is running in a context where the CI/CD workflow has copied certain files into the `frontend` directory. Because `rm` fails if files do not exist, when you run the build locally (assuming you didn't also needlessly copy the files in question), the build will fail.

## Description

Add `-f` to this line in `frontend/Dockerfile`:

```dockerfile
RUN rm ./frontend/.npmrc ./frontend/.pnpmfile.cjs ./frontend/pnpm-lock.yaml
```

should read:

```dockerfile
RUN rm -f ./frontend/.npmrc ./frontend/.pnpmfile.cjs ./frontend/pnpm-lock.yaml
```

## Alternatives

Change the way the CI/CD workflow runs so that it doesn't rely on these files being moved around at all. Then we can remove this CI implementation detail from the Dockerfile entirely, which ideally wouldn't know anything about it.

To do this, we probably need to more heavily leverage `pnpm`'s workspace stuff, but I'm not sure. It's almost certainly a lot more work to untangle the assumptions there.

**Comment by dhruvkb** [comment]
Historical context for why files are moved around in CI: https://github.com/WordPress/openverse/pull/446.

In short:
- Nuxt doesn't work well without shamefully-hoist
- `pnpm deploy` doesn't respect shamefully-hoist
- Replicating the workspace-like layout inside Docker needs files from outside the Docker context
- They can be brought into the context by `cp` before Docker build (which CI does)

A simpler approach, for now, might be to have a `build-img` recipe for the frontend that can do copy the files from the root, build the Docker image and then clean those files up once built.

**Comment by dhruvkb** [comment]
Recently learned about additional build contexts that remove the need for this file-juggling, implemented #2048 to resolve this issue.
